### ISSUE TITLE
Reset current branch - tab order

### ISSUE THREAD

**Issue by mstv** [gitextensions/gitextensions#7481]
## Current behaviour

On open, no focus is visible. The tab order is almost random or opposite.

## Expected behaviour

The pre-selected radion button is focused on open.
Logical tab order.

## Steps to reproduce

- `Reset current branch to here...`
- hit `Tab` 4x
- hit `Down` (sic!) several times

## Screenshots

![ResetCurrentBranch](https://user-images.githubusercontent.com/36601201/70053626-22d83200-15d6-11ea-8619-a24064ac1fcb.gif)

## Did this work in previous version of GitExtensions

at least not since the recent extension

## Environment

- Git Extensions 3.3.0
- Build 69763de058580d0dbf84be9a2d725b1719f607c6
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4018.0
- DPI 96dpi (no scaling)
